helm-chart/binderhub/templates/deployment.yaml

```diff
@@ -44,9 +42,6 @@ spec:
       {{- end }}
       volumes:
       - name: config
-        configMap:
-          name: binder-config
-      - name: secret-config
```

Comment:
I removed one volume definition related to the configmap, but let the volume be named config.